src/test/ui/symbol-names/trait-objects.legacy.stderr

```diff
@@ -1,53 +1,53 @@
-error: symbol-name(_ZN136_$LT$$RF$dyn$u20$core..ops..function..FnMut$LT$$LP$$RF$u8$C$$RP$$GT$$u2b$Output$u20$$u3d$$u20$$LP$$RP$$u20$as$u20$trait_objects..Bar$GT$6method17h1e14a5f2d365272fE)
-  --> $DIR/trait-objects.rs:15:5
+error: symbol-name(_ZN136_$LT$$RF$dyn$u20$core..ops..function..FnMut$LT$$LP$$RF$u8$C$$RP$$GT$$u2b$Output$u20$$u3d$$u20$$LP$$RP$$u20$as$u20$trait_objects..Bar$GT$6metSYMBOL_HASH)
+  --> $DIR/trait-objects.rs:17:5
    |
 LL |     #[rustc_symbol_name]
    |     ^^^^^^^^^^^^^^^^^^^^
 
-error: demangling(<&dyn core::ops::function::FnMut<(&u8,)>+Output = () as trait_objects::Bar>::method::h1e14a5f2d365272f)
-  --> $DIR/trait-objects.rs:15:5
+error: demangling(<&dyn core::ops::function::FnMut<(&u8,)>+Output = () as trait_objects::Bar>::metSYMBOL_HASH::SYMBOL_HASH)
+  --> $DIR/trait-objects.rs:17:5
    |
 LL |     #[rustc_symbol_name]
    |     ^^^^^^^^^^^^^^^^^^^^
 
-error: demangling-alt(<&dyn core::ops::function::FnMut<(&u8,)>+Output = () as trait_objects::Bar>::method)
-  --> $DIR/trait-objects.rs:15:5
+error: demangling-alt(<&dyn core::ops::function::FnMut<(&u8,)>+Output = () as trait_objects::Bar>::metSYMBOL_HASH)
+  --> $DIR/trait-objects.rs:17:5
```

Comment:
Is it expected that this changed from `::method` to `::metSYMBOL_HASH`?

Reply:
Oh this file shouldn't be here currently...(mostly because I didn't think hard enough about what regex to use here to make it reproducible and we don't care about legacy here anyways)